images/build/GitHubActions.Dockerfile

```diff
@@ -0,0 +1,165 @@
+# Folders in the image which we use to build this image itself
```

Comment:
@kichalla is the goal of this image to improve the pull time in a GitHub Action workflow by basing it off of the same `buildpack-deps` image they're using, or are there additional benefits we're trying to get from it as well?

Reply:
Yes, the goal is improve to the pull time in a GitHub action workflow. This PR uses the same layers of buildpacks-deps that currently the GitHub agents. This image also does not install any platform SDKs (except Yarn and Hugo for now). Platform SDKs would be dyanmically installed(Python for now, node and .NET Core support will be added soon).